.github/workflows/e2e-tests.yml

```diff
@@ -30,12 +30,11 @@ on:
       apiKey:
         required: false
         description: Authorization key
-  # pull_request:
-  #   branches:
-  #     - master
   push:
     branches:
       - master
+      - release
+      - prerelease
```

Comment:
For now please just add Master @amagyar-iohk , I have added you in another ADR that describes how the release process will be, which branches we are going to have, etc.
Thats worth analising together and agreeing upon!

bs